package.json

```diff
@@ -2,7 +2,7 @@
   "name": "calculator",
```

Comment:
**Tests as part of CI**

I would recommend creating a workflow (or refactor [`linters.yml`](https://github.com/Ginohmk/calculator-UsingReact/blob/display-feature-upgrade/.github/workflows/linters.yml)) in order to execute tests in every PR. It will be a final check the changes are not breaking any functionality.

Reply:
I see what you meant by adding the testing to my Gitflow in the linter.yml to improve continuous integration. I will look into that